engine/class_modules/sc_demon_hunter.cpp

```diff
@@ -553,6 +553,7 @@ class demon_hunter_t : public player_t
     double t30_havoc_2pc_fury_tracker = 0.0;
     const spell_data_t* t30_vengeance_2pc_buff;
     const spell_data_t* t30_vengeance_4pc_buff;
+    double t30_vengeance_4pc_soul_fragments_tracker = 0;
```

Comment:
You will need to add this to demon_hunter_t::reset() to set it back to 0 on iteration reset